spec/jobs/activity_tracking_job_spec.rb

```diff
@@ -62,5 +62,30 @@
         )
       }.to change { Notification.count }.by(2)
     end
+
+    it 'creates notifications when a comment has mentions' do
```

Comment:
this doesn't belong here, the reason I know is because this PR doesn't touch ActivityTrackingJob.

This code is testing the implementation of `Comments#notify` and it should belong in the corresponding spec.